### PR TITLE
Fix startup script: don't use `exec` from cron job

### PR DESCRIPTION
**Pull request by edmofro**
Will solve the fact that dev and feature deployments are not coming online

### CODE REVIEW

File: scripts/bash/downloadEnvironmentVariables.sh [hunk 1]
```diff
@@ -19,9 +19,7 @@ for PACKAGE in $PACKAGES; do
     ENV_FILE_PATH=${DIR}/../../packages/${PACKAGE}/.env
 
     # checkout branch specific env vars, or dev as fallback, temporarily redirecting stderr
-    exec 3> /dev/stderr 2> /dev/null
```
Comment (edmofro) [inline scripts/bash/downloadEnvironmentVariables.sh:22]: It was actually the `exec` that was breaking things when run from a cron job

**Review by IgorNadj — APPROVED** [review]
Ah that's what it was! Good work finding that!

File: scripts/bash/downloadEnvironmentVariables.sh [hunk 2]
```diff
@@ -19,9 +19,7 @@ for PACKAGE in $PACKAGES; do
...
-    lpass show --notes ${PACKAGE}.${BRANCH}.env > ${ENV_FILE_PATH} || lpass show --notes ${PACKAGE}.dev.env > ${ENV_FILE_PATH}
-    exec 2>&3
+    lpass show --notes ${PACKAGE}.${BRANCH}.env 2> /dev/null > ${ENV_FILE_PATH} || lpass show --notes ${PACKAGE}.dev.env > ${ENV_FILE_PATH}
```
Comment (edmofro) [inline scripts/bash/downloadEnvironmentVariables.sh:22]: Stderr now redirected specifically for the first "branch specific" one, which often fails before we fall back to dev

**Comment by edmofro** [comment]
> Ah that's what it was! Good work finding that!

Thanks...you'd be amazed at how long I spent trying to nail it down 😅

**Comment by edmofro** [comment]
Tested on feature deployment, works :+1:

**Pull request merged**